src/rubrix/server/server.py

```diff
@@ -127,6 +128,26 @@ def configure_app_logging(app: FastAPI):
     version=str(rubrix_version),
 )
 
+
+def configure_telemetry(app):
+    TELEMETRY_MESSAGE = """
+Server telemetry is enabled by default when you run the server.
+If you want to disable it, you can configure the environment variable:
+
+$>export RUBRIX_ENABLE_TELEMETRY=0
+
+You can find more info in docs https://rubrix.readthedocs.io/en/releases-0.x/community/telemetry.html?highlight=telemetry#telemetry
```

Comment:
Here is the current message 

@dvsrepo

Reply:
Rubrix uses telemetry to report anonymous usage and error information.  

You can know more about what information is reported at https://rubrix.readthedocs.io/en/releases-0.x/community/telemetry.html?highlight=telemetry#telemetry

Telemetry is currently enabled. If you want to disable it, you can configure the environment variable before relaunching the server:

$>export RUBRIX_ENABLE_TELEMETRY=0


@frascuchon @davidberenstein1957 this would be my proposal, please review!

Reply:
The message is now updated. 

Could you please try to launch a local server instance to validate the format?